### PR TITLE
Fixing description length of some index.md files

### PR DESCRIPTION
**Pull request by nhussein11**
Hey there, 

When running the gh action, I found that some index.md files did not meet the requirement of having a description between 120-160 characters.

This PR aims to fix those issues so that the gh run can be successful.

### CODE REVIEW

**Comment by eshaben** [comment]
@nhussein11 it seems like some files from [this PR](https://github.com/papermoonio/polkadot-ecosystem-docs-draft/pull/25/files) are included here, but that PR is still in draft mode - do we want to release that stuff or not yet?

File: docs/build-on-polkadot/parachains/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Parachains
-description: A guide to learn and interact with some of the parachains on Polkadot ecosystem.
+description: A guide to learn and interact with some of the parachains on the Polkadot ecosystem, including system parachains and common good parachains
```
Comment (eshaben) [inline docs/build-on-polkadot/parachains/index.md:3]:
```suggestion
description: A guide to learn and interact with some of the parachains in the Polkadot ecosystem, including system parachains and common good parachains.
```

File: docs/dev-tools/zombienet/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: ZombieNet
-description: ZombieNet is a testnet for Substrate based networks that is used spawn and test ephemeral networks.
+description: ZombieNet is a TestNet for Substrate based networks that is used spawn and test ephemeral networks with different configurations
```
Comment (eshaben) [inline docs/dev-tools/zombienet/index.md:3]:
```suggestion
description: ZombieNet is a TestNet for Substrate-based networks that is used to spawn and test ephemeral networks with different configurations.
```

File: docs/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Polkadot Draft Ecosystem Docs
-description: Explore the information, tools, and tutorials needed to start building on top of Polkadot, a blockchain network protocol that provides parachains with shared security and interoperability using XCM. 
+description: Discover the resources, tools, and tutorials essential for building on Polkadot, a blockchain protocol offering shared security and seamless interoperability
```
Comment (eshaben) [inline docs/index.md:3]:
```suggestion
description: Discover the resources, tools, and tutorials essential for building on Polkadot, a blockchain protocol offering shared security and seamless interoperability.
```

File: docs/networks/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Networks
-description: A breakdown of all the different networks within Polkadot.
+description: A breakdown of all the different networks within the Polkadot protocol, including live and test networks and their respective parachains
```
Comment (eshaben) [inline docs/networks/index.md:3]:
```suggestion
description: A breakdown of all the different networks within the Polkadot protocol, including live and test networks and their respective parachains.
```

File: docs/networks/testnets/index.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: TestNets
-description: A breakdown of all the different TestNets within Polkadot.
+description: A breakdown of all the different TestNets within Polkadot protocol, including tutorials and guides on how to interact with them
```
Comment (eshaben) [inline docs/networks/testnets/index.md:3]:
```suggestion
description: A breakdown of all the different TestNets within the Polkadot protocol, including tutorials and guides on how to interact with them.
```

**Review by eshaben — COMMENTED** [review]
good catches! so excited about all of this!

**Comment by nhussein11** [comment]
> @nhussein11 it seems like some files from [this PR](https://github.com/papermoonio/polkadot-ecosystem-docs-draft/pull/25/files) are included here, but that PR is still in draft mode - do we want to release that stuff or not yet?

@eshaben  I created this branch from the main branch, so this analyzer is already on the main. I believe that the PR you mentioned is for further customization of this bot.

**Comment by nhussein11** [comment]
> @nhussein11 it seems like some files from [this PR](https://github.com/papermoonio/polkadot-ecosystem-docs-draft/pull/25/files) are included here, but that PR is still in draft mode - do we want to release that stuff or not yet?

Yeah, that's correct. We can wait to merge this one until we merge that PR

**Review by dawnkelly09 — APPROVED** [review]
This is great, Nico. Thanks!

**Pull request merged**